### PR TITLE
Netspeed code style

### PR DESCRIPTION
**Pull request by raveit65**
- tabs to spaces
- start a new line with 4 spaces at the beginning

### CODE REVIEW

File: netspeed/src/netspeed.c [hunk 1]
```diff
@@ -1145,104 +1160,126 @@ static void
...
+                     gtk_builder_get_object (builder, "show_quality_icon_checkbutton"),
+                     "active", G_SETTINGS_BIND_DEFAULT);
+
+    g_settings_bind (applet->gsettings, "change-icon",
+                     gtk_builder_get_object (builder, "change_icon_checkbutton"),
+                     "active", G_SETTINGS_BIND_DEFAULT);
+
+    /* Default means device with default route set */
+    gtk_combo_box_text_append_text (GTK_COMBO_BOX_TEXT (applet->network_device_combo),
+                                                        _("Default"));
```
Comment (vkareh) [inline netspeed/src/netspeed.c:1208]: This indent is wrong, the `_("Default")` should be under the opening parenthesis of `gtk_combo_box_text_append_text`:
```
    gtk_combo_box_text_append_text (GTK_COMBO_BOX_TEXT (applet->network_device_combo),
                                    _("Default"));
```

File: netspeed/src/netspeed.c [hunk 2]
```diff
@@ -1145,104 +1160,126 @@ static void
...
+    ptr = devices = get_available_devices ();
+    for (i = 0; ptr; ptr = g_list_next (ptr)) {
+        gtk_combo_box_text_append_text (GTK_COMBO_BOX_TEXT (applet->network_device_combo),
+                                                            ptr->data);
```
Comment (vkareh) [inline netspeed/src/netspeed.c:1212]: Same here

**Comment by raveit65** [comment]
@vkareh 
All done.

**Review by vkareh — APPROVED** [review]
LGTM

File: netspeed/src/netspeed.c [hunk 3]
```diff
@@ -938,63 +944,64 @@ update_applet (MateNetspeedApplet *applet)
...
+                                                          GTK_DIALOG_MODAL,
+                                                          GTK_MESSAGE_ERROR,
+                                                          GTK_BUTTONS_OK,
+                                                          _("There was an error displaying help:\n%s"),
+                                                          error->message);
+        g_signal_connect (error_dialog, "response",
+                          G_CALLBACK (gtk_widget_destroy), NULL);
+
+        gtk_window_set_resizable (GTK_WINDOW (error_dialog), FALSE);
+        gtk_window_set_screen  (GTK_WINDOW (error_dialog), gtk_widget_get_screen (dialog));
```
Comment (rbuj) [inline netspeed/src/netspeed.c:1001]: two consecutive whitespace: `...screen  (GTK...`

File: netspeed/src/netspeed.c [hunk 4]
```diff
@@ -1009,132 +1016,140 @@ help_cb (GtkAction          *action,
...
+        "Perberos <perberos@gmail.com>",
+        NULL
+    };
+
+    gtk_show_about_dialog (NULL,
+                           "title", _("About MATE Netspeed"),
+                           "version", VERSION,
+                           "copyright", _("Copyright \xc2\xa9 2002-2003 Jörgen Scheibengruber\n"
+                                       "Copyright \xc2\xa9 2011-2014 Stefano Karapetsas\n"
+                                       "Copyright \xc2\xa9 2015-2020 MATE developers"),
```
Comment (rbuj) [inline netspeed/src/netspeed.c:1038]: quotation marks before `"Copyright \xc2\xa9...` should be aligned

File: netspeed/src/netspeed.c [hunk 5]
```diff
@@ -1529,216 +1568,218 @@ update_tooltip (MateNetspeedApplet* applet)
 
 
 static gboolean
-mate_netspeed_enter_cb(GtkWidget *widget, GdkEventCrossing *event, gpointer data)
+mate_netspeed_enter_cb (GtkWidget *widget,
+                        GdkEventCrossing *event,
+                        gpointer data)
```
Comment (rbuj) [inline netspeed/src/netspeed.c:1573]: on the other functions the name of the arguments are aligned

File: netspeed/src/netspeed.c [hunk 6]
```diff
@@ -1529,216 +1568,218 @@ update_tooltip (MateNetspeedApplet* applet)
...
 
-	return TRUE;
+    return TRUE;
 }
 
 static gboolean
-mate_netspeed_leave_cb(GtkWidget *widget, GdkEventCrossing *event, gpointer data)
+mate_netspeed_leave_cb (GtkWidget *widget,
+                        GdkEventCrossing *event,
+                        gpointer data)
```
Comment (rbuj) [inline netspeed/src/netspeed.c:1586]: same here

**Comment by rbuj** [comment]
@raveit65 you can also remove two consecutive empty blank lines:
```
$ find netspeed -name *.c -or -name *.h -exec sed -i -e '/./b' -e :n -e 'N;s/\n$//;tn' {} +
```

**Comment by raveit65** [comment]
@rbuj 
All done. Thx for the hint :)

**Review by rbuj — APPROVED** [review]
LGTM

**Pull request merged**